apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -5,11 +5,18 @@ import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className?: string;
 	to: ValueOf<typeof AppRoute>;
```

Comment:
Let's have optional props go last
```suggestion
	children: React.ReactNode;
	to: ValueOf<typeof AppRoute>;
	className?: string;
```